playbooks/charm/build.yaml

```diff
@@ -28,11 +28,20 @@
         files:
           - "{{ charm_build_name }}-{{ zuul.buildset }}.tar.bz2"
         delete_after: "14"
-      tasks:
-        - zuul_return:
-          data:
-            zuul:
-              artifacts:
-                - name: built charm tarball
-                  type: charm
-                  url: http://10.245.161.162:80/swift/v1/zuul/artifacts/built_charms/{{ charm_build_name }}-{{ zuul.buildset }}.tar.bz2
+      register: upload_results
+
+- block:
```

Comment:
aren't you missing a `hosts: all` or so as on line 1?

Reply:
oooh good catch, thanks!